hera_qm/tests/test_xrfi.py

```diff
@@ -26,22 +28,30 @@ def get_accuracy(f, rfi, verbose=VERBOSE):
         print '\t Found RFI: %1.3f\n\t False Positive: %1.3f' % (correctly_flagged, false_positive)
     return correctly_flagged, false_positive
 
+def fake_flags():
+    RFI = 10
```

Comment:
fake_flags is used in test_xrfi_simple (line 130) and test_watershed (line 145), this tests a failure mode(and coverage) of handing them an incorrect flag array and receiving an exceptionally low correct number of flags and high false positives.